Task_class_1/main.py

```diff
@@ -0,0 +1,69 @@
+def loops():
+    print('Insert a number to calculate squares: ')
+    try:
+        number = int(input())
+        for i in range(number):
+            print(f'{i ** 2}')
+    except ValueError:
+        print('Please enter a valid integer number')
+
+
+def listComprehension():
+    try:
+        print('Insert a number for the X Axis: ')
+        axisX = int(input())
+        print('Insert a number for the Y Axis: ')
+        axisY = int(input())
+        print('Insert a number for the Z Axis: ')
+        axisZ = int(input())
+        print('Insert a number to avoid in the Axes sum')
+        sumFilter = int(input())
+        ListOfNumbers = []
+        for x in range(axisX+1):
+            for y in range(axisY+1):
+                for z in range(axisZ+1):
+                    if (x+y+z) != sumFilter:
+                        ListOfNumbers.append([x, y, z])
```

Comment:
nice, now please try doing this using list comprehension https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions

Reply:
translated to listComprehension:
`[[x, y, z] for x in range(axisX+1) for y in range(axisY+1) for z in range(axisZ+1) if sumFilter != (x + y + z)]`